pywinauto/recorder/win32/win32_recorder.py

```diff
@@ -0,0 +1,216 @@
+import threading
+import ctypes
+import timeit
+import time
+
+from ..recorder_defines import EVENT, PROPERTY
+
+from ... import win32_hooks
+from ... import win32defines
+from ... import handleprops
+from ...win32structures import POINT
+from ...win32_element_info import HwndElementInfo
+from ...controls.hwndwrapper import InvalidWindowHandle
+from win32api import LOWORD, HIWORD
+import win32gui
+
+from ..uia.uia_recorder import ProgressBarDialog
+from ..control_tree import ControlTree
+from ..base_recorder import BaseRecorder
+from ..recorder_defines import RecorderEvent, \
+    RecorderKeyboardEvent, \
+    RecorderMouseEvent, \
+    ApplicationEvent, \
+    PropertyEvent
+
+from .injector import Injector
+from .common_controls_handlers import resolve_handle_to_event
+
+msg_id_to_key = {getattr(win32defines, attr_name): attr_name for attr_name in dir(win32defines) if attr_name.startswith('WM_') or attr_name.startswith('CB_')}
+
+def print_winmsg(msg):
+    pass
+    #print(handleprops.classname(msg.hWnd))
+    print("message:{}".format((msg_id_to_key[msg.message] if msg.message in msg_id_to_key else str(msg.message))))
```

Comment:
Could be used with default value: `msg_id_to_key.get(msg.message, str(msg.message))`